src/users_db.rs

```diff
@@ -173,9 +172,9 @@ impl UserBuilder {
             self.error = Some(UserBuilderError::InvalidPassword);
             return self;
         }
-        let mut md5 = Md5::new();
-        md5.input_str(&escape(&password));
-        self.password = md5.result_str();
+        if let Ok(h) = bcrypt::hash(&password) {
+            self.password = h;
+        }
```

Comment:
Currently we don't change the password in the object in the case of ````Err(_)````.  Not ideal but I didn't see an obvious alternative.